ntc_templates/templates/cisco_ios_show_interface_transceiver.textfsm

```diff
@@ -1,11 +1,14 @@
 Value INTERFACE (\w+\d+\/\S+)
 Value TEMPERATURE (\d+.\d+)
 Value VOLTAGE (\d+.\d+)
+Value CURRENT (\d+.\d+)
 Value TX_PWR (\S+)
 Value RX_PWR (\S+)
 
 Start
-  ^${INTERFACE}\s+${TEMPERATURE}\s+${VOLTAGE}\s+${TX_PWR}\s+${RX_PWR}\s+ -> Record
+  ^${INTERFACE}\s+${TEMPERATURE}\s+${VOLTAGE}\s+${CURRENT}?\s+${TX_PWR}\s+${RX_PWR}\s? -> Record
```

Comment:
Just in case I'd like to see `\s*` used that way we can have zero or more whitespace. (That is zero, one, two, etc.)

I'm going to push this one in as it is something regularly seen in the templates. Thank you.

```suggestion
  ^${INTERFACE}\s+${TEMPERATURE}\s+${VOLTAGE}\s+${CURRENT}?\s+${TX_PWR}\s+${RX_PWR}\s* -> Record
```